unity-client/Assets/Scripts/MainScripts/DCL/Components/BaseDisposable.cs

```diff
@@ -11,7 +11,7 @@ public abstract class BaseDisposable : IComponent
     {
         public virtual string componentName => GetType().Name;
         public string id;
-        public IParcelScene scene { get; protected set; }
+        public IParcelScene scene { get; set; }
```

Comment:
The setter had to be made public because the `scene` is now injected from outside in `ParcelScene`. The best implementation is to have an `Initialize(scene, id)`. I'm having this implemented in a continuation PR.